### PR TITLE
Adding ability use mail as remind method

### LINKED ISSUE #63: Add ability for send mail with links

### PR DESCRIPTION
**Pull request by esikachev**
Now you can use mail for getting links from request
close #63

### CODE REVIEW

**Comment by esikachev** [comment]
@ivanovmi @kozlovpn @dshakhray please, review this patch. I wrote it quickly, without going into details. maybe a lot of mistakes

File: shellper/base.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 from apiclient.discovery import build
 import argparse
+import base64
+from email.mime.text import MIMEText
```
Comment (esikachev) [inline shellper/base.py]: need import module, not class

**Comment by dshakhray** [comment]
Please fix readme.rst

**Comment by esikachev** [comment]
@ivanovmi @dshakhray @kozlovpn UPDATE

File: README.rst [hunk 2]
```diff
@@ -18,14 +18,15 @@ integration with Google Calendar.
...
+    * Integration with Google Calendar and Gmail
     * Adding tasks
     * Deleting tasks
     * Showing tasks
     * Searching information based on the title of tasks
 
 In this version authentication works with our test account. If you want that
-application works with your account, please delete file etc/calendar-api.json.
+application works with your account, please delete files etc/calendar-api.json and
+etc/gmail-api.conf.
```
Comment (ivanovmi) [inline README.rst]: Maybe `etc/gmail-api.json`?
And how about add description how to use `etc/shellper/shellper.conf` ?

Comment (esikachev) [inline README.rst]: @ivanovmi good point. this is will be added

File: shellper/base.py [hunk 3]
```diff
@@ -67,17 +79,33 @@ def authentication(self):
...
                 credentials = oauth2client.tools.run_flow(flow, store, flags)
-            else:
-                credentials = oauth2client.tools.run(flow, store)
             print 'Storing credentials to %s' % CREDENTIALS_PATH
         return credentials
 
+    def create_mail(self, config, to):
+        self.add_links(config)
+        links = config.get('description')[0]
+        body = '\n'.join([str(link) for link in links])
```
Comment (ivanovmi) [inline shellper/base.py]: What if returned value is `None`? Or blank string?

Comment (esikachev) [inline shellper/base.py]: @ivanovmi we will have empty mail

**Pull request merged**